src/ScanItems.js

```diff
@@ -224,17 +233,21 @@ class ScanItems extends React.Component {
   }
 
   checkout = (barcode) => {
-    const { mutator: { checkout } } = this.props;
+    const { mutator: { checkout }, patronBlockOverriddenInfo } = this.props;
```

Comment:
Could you please split props destructuring to several line?

Reply:
Done